pkg/normalize/validate.go

```diff
@@ -302,7 +304,7 @@ func NormalizeAndValidateConfig(config *models.DNSConfig) (errs []error) {
 			}
 
 			// Canonicalize Targets.
-			if rec.Type == "CNAME" || rec.Type == "MX" || rec.Type == "NAPTR" || rec.Type == "NS" || rec.Type == "SRV" {
+			if rec.Type == "CNAME" || rec.Type == "DS" || rec.Type == "MX" || rec.Type == "NAPTR" || rec.Type == "NS" || rec.Type == "SRV" {
```

Comment:
I don't think ds should use this case. There is no hostname in the target fields.

Reply:
Cool, can you think of any other validation checks for DS?

Reply:
I can't think of any.  We generally don't check the individual fields for fear of locking us into specific constants, etc.